docs/frontend/translation.rst

```diff
@@ -85,19 +102,19 @@ the transifex-client can be used::
    locale.
 
 
-German Du/Sie
--------------
+4. German Du/Sie
+----------------
 
 Adhocracy is currently used mostly in Germany, i.e. in German language.
 Unfortunately, there are two variants of German, a formal (Sie) and an
 informal (Du) one.
 
-All translations should use the informal variant.  The script
-``bin/change_german_salutation`` can be used to convert informal
-translations to formal ones.  Note that you will need to extend that
-script whenever the translation changes. The common workflow for this
-is: Iteratively run the script, check the output and add new rules until
-everything is fine.
+All translations should use the informal variant, except on the site
+mein.berlin.de. We use the script ``bin/change_german_salutation`` to 
```

Comment:
All german translations on transifex should still be informal, even for mein.berlin.de.

Also: I would like to keep specific projects out of the general adhocracy documentation.